koku/api/settings/test/cost_groups/test_query_handler.py

```diff
@@ -76,6 +77,17 @@ def test_get_cost_groups_filters(self):
                     for item in data:
                         self.assertEqual(item.get(filter_option), filter_value)
 
+    def test_get_cost_groups_filter_cluster(self):
+        """Basic test to exercise the API endpoint"""
+        param = {"filter[cluster]": OCP_ON_GCP_CLUSTER_ID}
+        url = self.url + "?" + urlencode(param, quote_via=quote_plus)
+        with schema_context(self.schema_name):
+            response = self.client.get(url, **self.headers)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
It's not necessary to url encode the parameters.
```suggestion
        param = {"filter[cluster]": OCP_ON_GCP_CLUSTER_ID}
        with schema_context(self.schema_name):
            response = self.client.get(self.url, param, headers=self.headers)
        self.assertEqual(response.status_code, status.HTTP_200_OK)
```